### DISCUSSION TITLE
[Rancher2] How to set maxPods in RKE2 Cluster

### DISCUSSION THREAD

**Discussion by lightelfua** [rancher/rke2#3453] [Q&A]
Hi, how to set maxPods in RKE2 Cluster?
Thank you.

**Comment by jtravee** [comment]
Hi @lightelfua, could you please verify if you are talking about RKE2 in the context of Rancher Manager, or do you mean RKE2 standalone? This will help us give you better direction.

**Comment by brandond** [comment]
`--max-pods` is a [kubelet argument](https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/#:~:text=Number%20of%20Pods). You can pass through args to the kubelet via the `--kubelet-arg` flag or [config file](https://docs.rke2.io/install/install_options/install_options/#configuration-file) entry. 

```yaml
kubelet-arg:
- max-pods=110
```

**Reply by dmytrobabeshko** [reply]
This failed kubelet startup. Have to use `kubelet-arg: "max-pods=200"` instead.

**Reply by brandond** [reply]
right, sorry. Was working from mobile and mistyped.

**Comment by lightelfua** [comment]
Rancher Manager
Go to Cluster Management -> Edit Cluster -> Advanced -> Additional Kubelet Args -> --max-pods=220
